recipe/meta.yaml

```diff
@@ -24,14 +24,16 @@ requirements:
     - m2-patch  # [win]
   host:
     - python
-    - cffi 1.16
+    - cffi 1.16  # [py<313]
+    - cffi 1.17  # [py>312]
     - pip
-    - setuptools
+    - setuptools <69.0.0 
     - wheel
     - zstd {{ zstd_version }}
   run:
     - python
-    - cffi >=1.11
+    - cffi >=1.11  # [py<313]
+    - cffi >=1.17  # [py>312]
```

Comment:
Same thing here

Reply:
Same for this, I got those from here: https://github.com/indygreg/python-zstandard/blob/0.23.0/setup.py#L30-L34